### ISSUE TITLE
Should the template include a folder for assets?

### ISSUE THREAD

**Issue by brianegan** [brianegan/new_flutter_template#12]
Should the template create a `/assets` or `/images` folder to guide folks of where to put assets? Do folks have a go-to / bullet-proof folder structure they use to organize assets that they think might make sense in a template? Please let us know :)

- No
- Yes (please tell us the name of the folder you'd like to see!)

**Comment by brianegan** [comment]
No

**Comment by brianegan** [comment]
Yes

**Comment by rexthecoder** [comment]
Yes...This is a big headache to me, It was a pain for me to set up my assets every time I create a new project. I figure something out which is much useful. Generating my asset, it like a light speed with just one command, You can check more from here https://pub.dev/packages/r_flutter

🥨

**Comment by orestesgaolin** [comment]
Yes, I'm in favor of populating the asset folder with sample image

**Comment by brianegan** [comment]
@orestesgaolin Thanks! I hadn't actually thought of putting an image in there, just setting up the folder :) Could you please describe why you'd like to see an image as well?

**Comment by orestesgaolin** [comment]
> @orestesgaolin Thanks! I hadn't actually thought of putting an image in there, just setting up the folder :) Could you please describe why you'd like to see an image as well?

Some public domain or Flutter related image (Dash maybe?) could be a good choice in my opinion. I've just had an idea that there could be a different Dash for every release of Flutter :P 

If the image was there it would be easier to spot it as newcomer. It could use a folder variant of specifying assets in `pubspec.yaml` so that it would be obvious that they don't need to provide full image name in the assets.

```yaml
flutter:
  assets:
    - assets/images/
```

**Comment by brianegan** [comment]
@orestesgaolin Excellent, thanks for the additional details and participation in general :)

**Comment by escamoteur** [comment]
I also think this is a good idea. And also display this image in the created UI because so many starters struggle with the correct asset path

**Comment by hamza-imran75** [comment]
I'd love to see /assets

**Comment by Arkangel12** [comment]
> > @orestesgaolin Thanks! I hadn't actually thought of putting an image in there, just setting up the folder :) Could you please describe why you'd like to see an image as well?
> 
> Some public domain or Flutter related image (Dash maybe?) could be a good choice in my opinion. I've just had an idea that there could be a different Dash for every release of Flutter :P
> 
> If the image was there it would be easier to spot it as newcomer. It could use a folder variant of specifying assets in `pubspec.yaml` so that it would be obvious that they don't need to provide full image name in the assets.
> 
> ```yaml
> flutter:
>   assets:
>     - assets/images/
> ```

love @orestesgaolin idea, totally agree!

**Comment by sbis04** [comment]
This would be really nice to include the `/assets` folder by default in the template because it is a very common thing to have assets, even in a very basic Flutter app. Also, I agree with @escamoteur and @orestesgaolin, we should have an image and display it in the UI (so that beginners get the sense of how to specify assets paths).

**Comment by sfshaza2** [comment]
I'd also love to see /assets with a sample image of Dash or the Flutter logo under /assets/image, perhaps. Also, a comment in the pubspec linking to where on flutter.dev to learn more. This would really help get newbies onto the right track and, for more experienced devs, save some setup time.

**Comment by RobertBrunhage** [comment]
Yes populating with some kind of image can go a long way, looking at for example a new project with React you will get the react logo as an image that is used in the app. So I generally agree with @orestesgaolin

**Comment by Anurag-Pola** [comment]
Yes, totally agree with @orestesgaolin. It would help newbies to understand how to use assets in their projects. A folder named /assets would do. As the assets have come up maybe you can also show up using a different font as even they come under this and this could give new users an idea that assets are not only images but anything and fonts are something which everyone will want to change.

**Comment by alestiago** [comment]
Building on @orestesgaolin  I would appreciate if not only an image was included but maybe also how to deal with images in different resolutions and screen sizes appropriately. Need to be careful since this could be very restrictive or be deleted by everyone that used the template if not done properly.

**Comment by ibhavikmakwana** [comment]
Adding on the @alestiago's point, we should add the `2x`, `3x` structure if we include the images folder.

Also what can be the naming convention for the assets,

What I do is for an icon it starts with the `icSomeName`, for images `imgSomeName`, and so on.

**Comment by burhanrashid52** [comment]
We can load the image from the asset with simple animation. #18

**Comment by ibhavikmakwana** [comment]
Proposing a structure here.

```
assets/images/
assets/images/2x/
assets/images/3x/
```

Maybe we should also have a separate section that will have SVGs only.

`assets/images/svgs/`
